.buildkite/pipeline_ab.py

```diff
@@ -15,11 +15,17 @@
 )
 
 perf_test = {
-    "block": {
-        "label": "🖴 Block Performance",
+    "virtio-block": {
+        "label": "🖴 Virtio Block Performance",
         "test_path": "integration_tests/performance/test_block_ab.py",
         "devtool_opts": "-c 1-10 -m 0",
     },
+    "vhost-user-block": {
+        "label": "🖴 vhost-user Block Performance",
+        "test_path": "integration_tests/performance/test_block_vhost_user_ab.py",
+        "devtool_opts": "-c 1-10 -m 0",
+        "timeout_in_minutes": 40,
+    },
```

Comment:
This needs to be 

```suggestion
    "virtio-block": {
        "label": "🖴 Virtio Block Performance",
        "test_path": "integration_tests/performance/test_block_ab.py::test_block_performance",
        "devtool_opts": "-c 1-10 -m 0",
    },
    "vhost-user-block": {
        "label": "🖴 vhost-user Block Performance",
        "test_path": "integration_tests/performance/test_block_ab.py::test_block_vhost_user_performance",
        "devtool_opts": "-c 1-10 -m 0",
        "timeout_in_minutes": 40,
    },
```

now

Reply:
indeed, thanks!

Reply:
Sorry for not catching this earlier >.>